emmarin/cl/risc0_proofs/ptx/src/main.rs

```diff
@@ -7,16 +7,22 @@ fn main() {
     let PtxPrivate {
         ptx,
         input_cm_paths,
-        cm_root,
+        cm_roots,
         from,
         to,
     } = env::read();
 
     assert_eq!(ptx.inputs.len(), input_cm_paths.len());
-    for ((input, cm_path), zone_id) in ptx.inputs.iter().zip(input_cm_paths).zip(&from) {
+    for (((input, cm_path), zone_id), cm_root) in ptx
+        .inputs
+        .iter()
+        .zip(input_cm_paths)
+        .zip(&from)
+        .zip(&cm_roots)
+    {
```

Comment:
Don't need `from` here. Can use `input.zone_id`

Reply:
same reason as above, we don't have zone id in notes / input witnesses, although we could add it